README.md

```diff
@@ -1 +1,15 @@
-# samples-is
\ No newline at end of file
+# Welcome to WSO2 Identity Server samples-is Repository!
+This is where all the sample code repositories developed for various reasons are collected. Please use below template to add a new sample to the repository.
+```
+# <project title>.
+## Introduction / Use case
+Explained the basic use case of the project. Include examples, external links, public jiras and other helpful material is possible.
+
+## Applicable product version(s)
+Mentioned the list of applicable IS versions.
+
+## How to use
+Provide a step by step guide on using this sample. Include steps on building the project, applying to Identity Server, neccessary configurations, etc.
```

Comment:
Better to break this in to two.
How to configure
How to test 
------- Sample request and response / Or steps